app/helpers/reservation_helper.rb

```diff
@@ -14,21 +14,13 @@ def update_links(reservation)
     case reservation.status
     when Reservation::CREATED
       content_tag(:ul, class: 'p-0 d-flex mb-3') do
-        concat(button_to('Confirm', seance_reservation_path(
-                                      reservation.seance,
-                                      reservation,
-                                      status: Reservation::CONFIRMED
-                                    ),
+        concat(button_to('Confirm', confirm_reservation_path(reservation),
                          method: :patch, class: 'btn btn-success me-2'))
-        concat(button_to('Cancel', seance_reservation_path(
-                                     reservation.seance,
-                                     reservation,
-                                     status: Reservation::CANCELLED
-                                   ),
+        concat(button_to('Cancel', cancel_reservation_path(reservation),
```

Comment:
I think you don't need to use [concat](https://apidock.com/rails/ActionView/Helpers/TextHelper/concat) :thinking: 
Have you spotted a problem here?